HIDInjector/app/HidInjectorTest.c

```diff
@@ -154,9 +154,14 @@ BOOL SendTouch(HANDLE File,
         TouchState.Report.TouchReport.ContactCount = i == 0 ? count : 0; // Only the first report contains the contact count for the frame.
         TouchState.Report.TouchReport.ContactIndentifier = contacts[i].pointerInfo.pointerId;
 
-        //the value is expected to be normalized and clamped to[0, 65535]
-        TouchState.Report.TouchReport.AbsoluteX = (USHORT)max(0, min(contacts[i].pointerInfo.ptPixelLocation.x * GetSystemMetrics(SM_CXSCREEN), 0x7FFF));
-        TouchState.Report.TouchReport.AbsoluteY = (USHORT)max(0, min(contacts[i].pointerInfo.ptPixelLocation.y * GetSystemMetrics(SM_CYSCREEN), 0x7FFF));
+        //the value is expected to be normalized and clamped to[0, 32767]
+		//normalize factor -> physical resolution to screen resolution
+		float normX = (float)TOUCH_PHYSICAL_MAX / GetSystemMetrics(SM_CXSCREEN);
+		float normY = (float)TOUCH_PHYSICAL_MAX / GetSystemMetrics(SM_CYSCREEN);
+
+		//normalize and clamp
+        TouchState.Report.TouchReport.AbsoluteX = (USHORT)max(0, min(contacts[i].pointerInfo.ptPixelLocation.x * normX, 0x7FFF));
```

Comment:
number is still there. Please replace with macro TOUCH_PHYSICAL_MAX is that's the intent